packages/app/src/components/Posts.tsx

```diff
@@ -54,19 +53,13 @@ const Post: FC<PostProps> = ({ post }) => {
 				<p>{post.content}</p>
 			</div>
 			<Image src={post.imagelink} />
-			
 		</StyledPost>
 	);
 };
 
 export default function Posts() {
 	const { loading, error, data } = useAllPostsQuery();
 
-	/*const [createPostMutation] = useCreatePostMutation();
-  const handleCreatePost = () => {
-    createPostMutation({variables:{}})
-  }*/
```

Comment:
Should this be removed? Is perttu still working on this?

Reply:
no working